word-count/word_count_test.rb

```diff
@@ -0,0 +1,74 @@
+require 'minitest/autorun'
+require_relative 'word_count'
+
+class PhraseTest < Minitest::Test
+  def test_count_one_word
+    phrase = Phrase.new('word')
+    counts = { 'word' => 1 }
+    assert_equal counts, phrase.word_count
+  end
+
+  def test_count_one_of_each
+    phrase = Phrase.new('one of each')
+    counts = { 'one' => 1, 'of' => 1, 'each' => 1 }
+    assert_equal counts, phrase.word_count
+  end
+
+  def test_count_multiple_occurrences
+    phrase = Phrase.new('one fish two fish red fish blue fish')
+    counts = { 'one' => 1, 'fish' => 4, 'two' => 1, 'red' => 1, 'blue' => 1 }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.